doc/Versioning.rst

```diff
@@ -0,0 +1,15 @@
+Versioning
+##########
+
+The ``pyTooling.Versioning`` package provides auxiliary classes to implement
+`semantic <https://semver.org/>`__ and `calendar <https://calver.org/>`__ versioning.
+
+SemVersion
+**********
+
+The :class:`SemVersion` class represents of a version number like ``v3.7.12``.
```

Comment:
Because of changed Sphinx structure and enabled autoapi extension, modules got double documented. To compensate, `automodule` and `autoclass` directives got removed.

A complete rewrite of the documentation is necessary!!

Reply:
Moved to #42.